src/test/java/teammates/test/cases/logic/InstructorsLogicTest.java

```diff
@@ -522,13 +484,6 @@ public void testVerifyIsEmailOfInstructorOfCourse() throws Exception  {
         } catch (AssertionError e) {
             AssertHelper.assertContains("Supplied parameter was null", e.getMessage());
         }
-
-        try {
-            instructorsLogic.verifyIsGoogleIdOfInstructorOfCourse(instructorEmail, null);
```

Comment:
@tarun14110 this method was likely mistakenly tested here by previous developers, as it should be testing for `verifyIsEmailOfInstructorOfCourse` rather than `verifyIsGoogleIdOfInstructorOfCourse`. Let's fix this by testing `instructorsLogic.verifyIsEmailOfInstructorOfCourse(instructorEmail, null);`